css-positioning/absolute.css

```diff
@@ -0,0 +1,42 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url("images/one-does-not-simply.jpg");
+  background-size: cover;
+  position: relative;
+
+  /* your code here */
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: absolute;
+  right: 0.3em;
+  top: 0.3em;
+
+  /* your code here */
+}
+
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+  position: absolute;
+  top: 5em;
+
+  /* your code here */
+}
+
+.overlay {
+  background: linear-gradient(transparent, black);
+
```

Comment:
This still needs to be done.

Reply:
![image](https://user-images.githubusercontent.com/117887912/206805742-8c819106-282c-49c5-b078-70d2dfdb14f9.png)